docs/source/tutorials/artifact.rst

```diff
@@ -0,0 +1,258 @@
+
+
+
+Keys
++++++
+
+Artifacts store data under keys. Each key is of the form
+``<type>.<name>.<measure>``, e.g., "cause.all_causes.restrictions" or
+``<type>.<measure>``, e.g., "population.structure." To view all keys in an
+artifact, use the ``keys`` attribute of the artifact:
+
+.. code-block:: python
+
+    art.keys
+
+::
+
+    ['metadata.keyspace', 'metadata.locations', 'metadata.versions', 'population.age_bins',
+     'population.structure', 'population.theoretical_minimum_risk_life_expectancy']
+
+
+Reading data
+-------------
+
+Now that we've seen how to create an :class:`~vivarium.Artifact` object and
+view the underlying storage structure, let's cover how to actually retrieve
+data from that artifact. We'll use the :func:`~vivarium.Artifact.load` method.
+
+We saw the key names in our artifact in the previous step, and we'll use those
+names to load data. For example, if we want to load the population structure
+data from our Artifact we do:
+
+.. code-block:: python
+
+    art = Artifact('test_artifact.hdf')
+    pop = art.load('population.structure')
+    print(pop.head()))
+
+::
+
+                                                                           value
+    age_group_end age_group_start location sex    year_end year_start
+    0.019178      0.0             Ethiopia Female 2007     2006        25610.50
+                                           Male   2012     2011        29136.66
+                                                  2009     2008        27492.91
+                                           Female 2000     1999        22157.50
+                                                  1993     1992        19066.45
+
+Notice that if we construct our artifact with filter terms as discussed
+above, we'll filter the data that gets loaded out of it:
+
+.. code-block:: python
+
+    art = Artifact('test_artifact.hdf', filter_terms=['age_group_start > 5'])
+    pop = art.load('population.structure')
+    print(pop.head()))
+
+::
+
+                                                                            value
+    age_group_end age_group_start location sex    year_end year_start
+    15.0          10.0            Ethiopia Male   2011     2010        6009393.00
+                                                  2003     2002        4489336.99
+                                           Female 2016     2015        6424674.99
+                                           Male   2017     2016        6610845.00
+                                           Female 2006     2005        4922733.99
+
+We can only load keys that already exist in the Artifact, however. If we try
+to load a key not present in our Artifact, we will get an error:
+
+.. code-block:: python
+
+    art.load('a.fake.key')
+
+::
+
+    Traceback (most recent call last):
+      File "<stdin>", line 1, in <module>
+      File "/home/kate/code/vivarium/vivarium_public_health/src/vivarium_public_health/dataset_manager/artifact.py", line 75, in load
```

Comment:
This exception example still references the path in vph so we should probably update